pkg/server/container_create.go

```diff
@@ -341,6 +341,10 @@ func addOCIDevices(g *generate.Generator, devs []*runtime.Device, privileged boo
 				UID:   &hostDevice.Uid,
 				GID:   &hostDevice.Gid,
 			}
+			if hostDevice.Major == 0 && hostDevice.Minor == 0 {
```

Comment:
On second thought, given that:
1) Major 0 is never used`http://www.fsl.cs.sunysb.edu/kernel-api/re941.html`
2) I'd like to make the spec generation reusable library.

I'd prefer not to add log in this function. :)